init_sd_card/command.py

```diff
@@ -314,6 +314,26 @@ def step_flash(_, parsed, data):
                 dtslogger.info("Please retry while specifying a valid device. Bye bye!")
                 exit(4)
 
+    # Safety check to prevent large partition accidentally written
+    if sd_type == "SD":
+        try:
+            while True:
+                command = 'lsblk | grep '+ parsed.device.replace("/dev/","")
+                disk_info = os.popen(command).read().split()
```

Comment:
`lsblk` has machine-readable outputs which might be more stable and portable here, i.e. `lsblk -J` or `lsblk -P`.